src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -144,10 +150,23 @@
     @Option(name = "-solr.zkChroot", usage = "the ZooKeeper chroot if using SolrCloud")
     public String solrZkChroot = "/";
 
-    // Note: This is used for ConcurrentSolrClient, where each processing thread has its own instance
-    // If the architecture is changed to share one ConcurrentSolrClient among all threads, the default should likely be increased
-    @Option(name = "-solr.client.threads", metaVar = "[Number]", required = false, usage = "Number of Threads for each SolrClient")
-    public int solrClientThreads = 16;
+    /**
+     * The number of {@link SolrClient#add(Collection) add} calls to buffer before draining pending update queue.
+     */
+    @Option(name = "-solr.client.queue.size", usage = "the number of update requests to buffer before draining")
+    public int solrClientQueueSize = 16;
```

Comment:
Should we use standard Java camelCasing here? e.g., `solr.clientQueueSize`? `solr.` is fine I think because that's how we've grouped the options...